src/widgets/panel/sidepanelwidget.cpp

```diff
@@ -31,9 +31,11 @@ SidePanelWidget::SidePanelWidget(QPixmap* p, QWidget* parent)
     }
 
     QFormLayout* colorForm = new QFormLayout();
+    colorForm->setRowWrapPolicy(QFormLayout::WrapLongRows);
     m_toolSizeSlider = new QSlider(Qt::Horizontal);
     m_toolSizeSlider->setRange(1, maxToolSize);
     m_toolSizeSlider->setValue(m_toolSize);
+    m_toolSizeSlider->setMinimumWidth(100);
```

Comment:
Can you name this constant similar to maxtoolSize on line 36? Thanks for the contribution btw!

Reply:
You got it, thanks for the cool tool!